Marlin/src/lcd/language/language_en.h

```diff
@@ -862,6 +862,7 @@ namespace Language_en {
   LSTR MSG_DRIVER_BACKWARD                = _UxGT(" driver backward");
 
   LSTR MSG_SD_CARD                        = _UxGT("SD Card");
+  LSTR MSG_SDIO_CARD                      = _UxGT("SDIO Card");
```

Comment:
To most users, "SDIO" doesn't mean anything. It would therefore be better to refer to multiple SD card drives as "SD Card 1," "SD Card 2," etc.

Reply:
I think numbered SD cards would be doable. There can just be a hierarchy comment in the Configuration that the priority for numbering of storages is something like SD -> SDIO -> usbflash. And also a Configuration option that lets users define a custom name for each of their storage options to more clarity in cases where onboard SD may be SD and LCD is SDIO or usb flash. 

In `menu_media.cpp` for the ` menu_media_select()` menu, nested macros can define the correct numbering.